client/src/components/organisms/Home/Home.jsx

```diff
@@ -11,7 +12,8 @@ import VideoGroup from "./../VideoGroup/VideoGroup"
 import {searchCreator} from "./../../../utils/category/category"
 
 function Home() {
```

Comment:
Did you notice when you scroll down and click on a link, it doesn't scroll to the top of the next page?

Add a useEffect for every page
This might help:
`React.useEffect(() => {
    window.scrollTo(0, 0);
  }, []);`

Reply:
Thanks Rashita! This was very useful :)
Adding this for sure.